frontend/src/docs/MigrationImageBehaviour.tsx

```diff
@@ -0,0 +1,26 @@
+//Script to migrate from next/legacy/image to next/image
+//paddingBottom is controlled by a width to height ratio
+
+import Image from 'next/image'
+
+export function imageLoader({ src }: { src: string }) {
+  return src
+}
+
+function ResponsiveImage({ src, scaling = 50, alt }) {
+  return (
+    <div
+      style={{
+        position: 'relative',
+        width: 'auto',
+        height: 'auto',
+        marginLeft: 'auto',
+        paddingBottom: `max(350px, ${scaling}%)`,
+      }}
+    >
+      <Image loader={imageLoader} className='next-image' src={src} layout='fill' objectFit='contain' alt={alt} />
```

Comment:
According to [https://nextjs.org/docs/pages/api-reference/components/image-legacy#comparison](the docs), `layout` and `objectFit` are replaced with `style` and/or `className`.